web-terminal/instance/base/web-terminal-tooling.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: workspace.devfile.io/v1alpha2
+kind: DevWorkspaceTemplate
+metadata:
+  annotations:
+    controller.devfile.io/allow-import-from: '*'
+  name: web-terminal-tooling
+  namespace: openshift-operators
+  labels:
+    console.openshift.io/terminal: 'true'
+spec:
+  components:
+    - container:
+        args:
+          - tail
+          - '-f'
+          - /dev/null
+        cpuLimit: 400m
+        cpuRequest: 100m
+        image: registry.redhat.io/web-terminal/web-terminal-tooling-rhel8@sha256:657874adad93a9e6c2b746b836015de410d2e280f85c079c81e24ae81343fa6c
```

Comment:
Any way we can do what you are trying to do without pinning a specific image?

I suspect this is going to become out of date fairly quickly....

Reply:
I don't like this either. This is more of a reference to the original. I wanted this for reference. There are a few ways to do this... I've thought about doing a build config to actually build a custom image to run... Maybe that is a good option to explore